WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/RangedDatePicker.swift

```diff
@@ -30,16 +59,22 @@ struct RangedDatePicker: View {
     ///
     private let datesFormatter: RangedDateTextFormatter
 
+    /// Custom text for the confirm button
+    ///
+    private let applyButtonTitle: String
+
     /// Custom `init` to provide intial start and end dates.
     ///
     init(startDate: Date? = nil,
          endDate: Date? = nil,
          datesFormatter: RangedDateTextFormatter,
+         applyButtonTitle: String = Localization.apply,
```

Comment:
Instead of duplicating the `Localization.apply` string in `RangedDatePickerHostingController`, we can make this parameter optional with `nil` as the default value. 

Inside the `init`, if `applyButtonTitle` is not sent from outside, we can fallback to using `Localization.apply`.

What do you think?

Reply:
Great idea, I also renamed the parameter name for better clarity. Done in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/ef8a84cf51dc5bb63baa83248466600c093547e3